src/resources/deal21Cards.js

```diff
@@ -0,0 +1,8 @@
+const FIRST_CARD = 0;
+const DEAL_NUMBER = 21;
+
+function deal21Cards(deck) {
```

Comment:
O que acha de criar uma função mais genérica/reutilizável que tem como responsabilidade devolver um número x de cartas ao invés de ficar travado em 21?

Fica até mais simples pra testar, porque seus exemplos não precisam ser exatamente usando 21 cartas.
```suggestion
function dealCards(deck, quantity) {
```

Reply:
Neste caso, eu poderia chamar no componente o número 21 solto, ou seria mais adequado uma varivável contendo o valor 21?

```js
dealsCards(deck, 21)

or

dealsCards(deck, DEAL_NUMBER)
```

Reply:
Mais apropriado seria `dealsCards(deck, DEAL_NUMBER)` sendo que essa constante exista somente no componente.